hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -420,6 +423,25 @@ export class Datanodes extends React.Component<Record<string, object>, IDatanode
       showDataFetchError(error.toString());
     });
   };
+  
+  removeDatanode= async (selectedRowKeys: any) => {
+    try {
+      await axios.put('/api/v1/datanodes/remove', selectedRowKeys);
```

Comment:
Could we update this to use the axios helpers? Since this is a PUT operation it would be better we ensure the request is cancelled if we are changing context.

Reply:
@devabhishekpal 
Completed changes in latest commit and tested locally and attached screenshot for abort signal also